include/Basic/Tokens.hpp

```diff
@@ -1,7 +1,10 @@
 #ifndef GLU_TOKENS_HPP
 #define GLU_TOKENS_HPP
 
+#include <ostream>
```

Comment:
don't include ostream in a header. we use llvm::outs() and raw_output_stream, not std::cout and expensive <ostream>